docs/pages/web/tabs.js

```diff
@@ -25,8 +25,8 @@ function DefaultExample() {
       activeTabIndex={activeIndex}
       onChange={({ activeTabIndex }) => { setActiveIndex(activeTabIndex); }}
       tabs={[
-        { href: "#", text: "Explore", indicator: "dot" },
-        { href: "#", text: "Shop" },
+        { href: "#", text: "Explore" },
+        { href: "#", text: "Shop", indicator: "dot" },
```

Comment:
We still want to show what the component is capable of in this top example, but I want to make it more clear that the dot and underline are not indicating the same thing